api/v1beta1/activemqartemis_types.go

```diff
@@ -412,4 +412,11 @@ const (
 	DeployedConditionNotReadyReason  = "PodsNotReady"
 	DeployedConditionZeroSizeReason  = "ZeroSizeDeployment"
 	DeployedConditionZeroSizeMessage = "Pods not scheduled. Deployment size is 0"
+
+	BrokerPropsConditionType                             = "BrokerProperties"
```

Comment:
naming is hard!

I think maybe the condition is BrokerPropertiesInSync or BrokerPropertiesApplied, actually think Applied will help in the future when the status contains errors on failure to apply.

Applied = True -> in sync and no errors

Reply:
Applied = Unknown for older broker that does not expose Status or when no jolokia agent yet ready

Reply:
I think any Unknown should filter up to the  (total) Ready condition being unknown

Reply:
Changes applied, I am having some issues with the IT, don't merge yet